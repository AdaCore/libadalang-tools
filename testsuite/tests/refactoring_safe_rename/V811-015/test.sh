safe_rename -P default.gpr -S com_channel.adb -L 7 -R 17 -N Status --algorithm analyse_ast
