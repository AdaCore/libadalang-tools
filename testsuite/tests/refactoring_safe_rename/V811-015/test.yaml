description: |
    This test checks if the safe rename tool Name_Hidden_Finder ignores
    references of actual parameters on parameter associations.

driver: shell_script
