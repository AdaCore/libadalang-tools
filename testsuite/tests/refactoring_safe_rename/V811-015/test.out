# Renaming node com_channel.adb:7:16: 
References:
Unit:com_channel.adb
Line: 7
Source Location Range:
Start_Line: 7
Start_Column: 16
End_Line: 7
End_Column: 22
Unit:neutrino.adb
Line: 2
Source Location Range:
Start_Line: 2
Start_Column: 23
End_Line: 2
End_Column: 29
Line: 4
Source Location Range:
Start_Line: 4
Start_Column: 7
End_Line: 4
End_Column: 13
Unit:neutrino.ads
Line: 2
Source Location Range:
Start_Line: 2
Start_Column: 23
End_Line: 2
End_Column: 29

